sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/GraphClient.java

```diff
@@ -50,11 +50,13 @@ public GraphClient(AADAuthenticationProperties properties) {
                 LOGGER.error("Can not get group information from graph server.", ioException);
                 break;
             }
-            memberships.getValue()
-                       .stream()
-                       .filter(this::isGroupObject)
-                       .map(Membership::getDisplayName)
-                       .forEach(groups::add);
+            if(!properties.getUserGroup().getEnableFullList()){
```

Comment:
will it  have  npe here ?

Reply:
The default value of enableFullList is false